Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -213,6 +213,28 @@ - (void)testReportWithAppCrashingTestsSet {
     [self assertGotReport:junitReportPath isEqualToWantReport:expectedFilePath];
 }
 
+- (void)testSaveRetryXml {
+    NSString *testBundlePath = [BPTestHelper sampleAppCrashingTestsBundlePath];
```

Comment:
I don't see any retry with any tests in this bundle

Reply:
That bundle is also used for testReportWithAppCrashingAndRetryOnlyFailedTestsSet, which retries as the name implies, which is why I grabbed it to test this as well.